notebooks/official/gen_ai/batch_eval_llm.ipynb

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            project_number = shell_output[-1].split(":")[1].strip().replace("'", "")
</pre><p> </p><p>Use <code>--format</code> option instead of <code>shell_output</code></p><p> </p><p><code>gcloud projects describe $PROJECT_ID --format="value(projectNumber)"</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
done